src/plugins/intel_gna/tests/unit/ops/util_test.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstdint>
+#include <vector>
+#include <thread>
+#include <memory>
+
+#include <gtest/gtest.h>
+#include "ops/util/util.hpp"
+
+using namespace ov::intel_gna::ngraph_util;
+
+namespace test {
+
+using GnaOpsUtilTestParams = std::tuple<std::shared_ptr<ngraph::Node>, bool>;
+
+class GnaOpsUtilTest : public ::testing::TestWithParam<GnaOpsUtilTestParams> {
+public:
+    static std::string get_test_name(const testing::TestParamInfo<GnaOpsUtilTestParams>& obj) {
+        std::shared_ptr<ngraph::Node> node;  // node to be converted
+        bool result; // expected result
+        std::tie(node, result) = obj.param;
+
+        std::ostringstream test_name;
+        test_name << "Node=" << node->get_type_info().name << "_";
+        test_name << "Op=" << node->get_type_info().get_version() << "_";
+        test_name << "ExpectedReulst=" << result;
+
+        return test_name.str();
+    }
+};
+
+class GnaOpsUtilisPoolingTest : public GnaOpsUtilTest {
+public:
+    void validate() {
+        std::shared_ptr<ngraph::Node> node;  // node to be converted
+        bool result; // expected result
+        std::tie(node, result) = GetParam();
+        ASSERT_TRUE(result == is_pooling(node));
+    }
+};
+
+class GnaOpsUtilIsEltwiseMulTest : public GnaOpsUtilTest {
+public:
+    void validate() {
+        std::shared_ptr<ngraph::Node> node;  // node to be converted
+        bool result; // expected result
+        std::tie(node, result) = GetParam();
+        ASSERT_TRUE(result == is_eltwise_mul(node));
+    }
+};
+
+class GnaOpsUtilIsEltwiseAddTest : public GnaOpsUtilTest {
+public:
+    void validate() {
+        std::shared_ptr<ngraph::Node> node;  // node to be converted
+        bool result; // expected result
+        std::tie(node, result) = GetParam();
+        ASSERT_TRUE(result == is_eltwise_add(node));
+    }
+};
+
+TEST_P(GnaOpsUtilisPoolingTest, isPoolingTest) {
+    validate();
+}
+
+TEST_P(GnaOpsUtilIsEltwiseMulTest, isEltwiseMulTest) {
+    validate();
+}
+
+TEST_P(GnaOpsUtilIsEltwiseAddTest, isEltwiseAddTest) {
+    validate();
+}
+
+ov::NodeVector pooling_nodes_false = {
+    std::make_shared<ngraph::op::VariadicSplit>(),
+    std::make_shared<ngraph::op::Concat>(),
+    std::make_shared<ngraph::op::MatMul>(),
+    std::make_shared<ngraph::opset9::MaxPool>()
+};
+
+ov::NodeVector pooling_nodes_true = {
+    std::make_shared<ngraph::opset7::MaxPool>()
+};
+
+ov::NodeVector elwise_mul_nodes_false = {
+    std::make_shared<ngraph::op::VariadicSplit>(),
+    std::make_shared<ngraph::op::Eltwise>(ngraph::Output<ov::Node>(std::make_shared<ngraph::opset7::Constant>()),
```

Comment:
The same situation with ngraph::Output. Consider using ov::Output instead of ngraph::Output or just add
using namespace ov;
using namespace ov::opset9;

Reply:
the problem that ov::opset9 is missing

Reply:
Even if add #include "openvino/opsets/opset9.hpp" ?